lib/rules/selector-max-attribute/README.md

```diff
@@ -75,3 +75,25 @@ The following patterns are *not* considered violations:
 /* `[disabled]` is inside `:not()`, so it is evaluated separately */
 [type="text"][name="message"]:not([disabled]) {}
 ```
+
+## Optional secondary options
+
+### `ignoreAttributes: ["/regex/", "string"]`
+
+Given:
+
+```js
+["/^my-/", "dir"]
+```
+
+For example, with `2`.
```

Comment:
"For example, with `0`."

With `2` it wouldn't be considered a violation as there are only two attributes selectors in the example.